cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3862,6 +3916,17 @@ TEST(TestArrowReaderAdHoc, CorruptedSchema) {
   TryReadDataFile(path, ::arrow::StatusCode::IOError);
 }
 
+TEST(TestArrowParquet, LargeByteArray) {
```

Comment:
This test need to be wrapped in `#ifdef ARROW_WITH_BROTLI` as some builds may not enable brotli.